Staking / Ursula guide update (#1284)
nucypher/nucypher: docs/source/guides/ursula_configuration_guide.rst

.. _ursula-config-guide:

==========================
Ursula Configuration Guide
==========================

1. Install NuCypher and geth
------------------------------

> If you want to run a NuCypher node which participates in decentralized network,

Reviewer: ```suggestion
If you want to run a NuCypher node that participates in the decentralized network,
```

you need to install it first. The installation procedure for the Ursula (Worker)
node is exactly the same as for Staker.

First, you install geth. You'll need to run it in the background and sync up.
For testnet, it is:

.. code:: bash

    $ geth --goerli --no-usb

You need to create a software-controlled account in geth:

.. code:: bash

    $ geth attach ~/.ethereum/goerli/geth.ipc
    > personal.newAccount();
    > eth.accounts[0]
    ["0xc080708026a3a280894365efd51bb64521c45147"]

So, your worker account is ``0xc080708026a3a280894365efd51bb64521c45147`` in
this case.

Before installing ``nucypher``, you may need to install necessary developer
tools and headers, if you don't have them already. In Ubuntu, Debian, Linux Mint
or similar distros, that is:

.. code:: bash

    $ sudo apt install build-essential python3-dev python3-pip

Install ``nucypher`` either by doing ``sudo pip3 install nucypher`` if you have
a dedicated instance or container, or with a ``virtualenv``:

.. code:: bash

    $ virtualenv -p python3 nucypher
    $ source nucypher/bin/activate
    (nu)$ pip3 install nucypher

> Before continuing, Verify your ``nucypher`` installation and entry points are functional:

Reviewer 2: ```suggestion
Before continuing, verify that your ``nucypher`` installation and entry points are functional:
```

> Activate your virtual environment (if you already didn't) and run the ``nucypher --help`` command

Reviewer 2: ```suggestion
Activate your virtual environment (if you haven't already) and run the ``nucypher --help`` command
```
